theatre/studio/src/panels/SequenceEditorPanel/layout/tree.ts

```diff
@@ -16,12 +16,31 @@ import logger from '@theatre/shared/logger'
 import {titleBarHeight} from '@theatre/studio/panels/BasePanel/common'
 import type {Studio} from '@theatre/studio/Studio'
 
-export type SequenceEditorTree_Row<Type> = {
-  type: Type
+/**
+ * Base "view model" for each row with common
+ * required information such as row heights & depth.
+ */
+export type SequenceEditorTree_Row<TypeName extends string> = {
+  /** type of this row, e.g. `"sheet"` or `"sheetObject"` */
+  type: TypeName
+  /** Height of just the row in pixels */
   nodeHeight: number
+  /** Height of the row + height with children in pixels */
   heightIncludingChildren: number
+
+  /** Visual indentation */
   depth: number
+  /**
+   * This is a part of the tree, but it is not rendered at all,
+   * and it doesn't contribute to height.
+   */
+  shouldRender: boolean
```

Comment:
did you consider, instead of adding this property, to just _not_ add leafs that would have `shouldRender = false`?

Reply:
I think I am just missing something. Did you use the tree to calculate aggregates?